odml/tools/xmlparser.py

```diff
@@ -40,8 +44,14 @@ def to_csv(val):
 def from_csv(value_string):
     if not value_string:
         return []
-    if value_string[0] == "[":
+    if value_string[0] == "[" and value_string[-1] == "]":
         value_string = value_string[1:-1]
+    else:
+        # This is a single string entry, any comma contained
+        # is part of the value and must not be used to
+        # split up the string.
+        return [value_string]
+
```

Comment:
:+1:

Reply:
@JuliaSprenger caught that one, I'm just taking it one step further. ;)